modules/diff.js

```diff
@@ -1,72 +1,66 @@
 const axios = require('axios')
 const NetlifyAPI = require('netlify')
 
-const { lambda, getFuncName } = require('./util')
+const { userInGroup, invokeSlackWorker, errMsg } = require('./util')
 
-const { GITHUB_TOKEN, COMMIT_LIMIT = 10, NETLIFY_TOKEN, DEPLOYED = false } = process.env
+const { GITHUB_TOKEN, COMMIT_LIMIT = 50 - 2, NETLIFY_TOKEN, DEPLOYED = false } = process.env
```

Comment:
is there a reason why the limit is `50 - 2` instead of `48`?

Reply:
50 is the number of blocks allowed in [Slack Block Kit](https://api.slack.com/reference/block-kit/blocks). 2 means that there are 2 staple blocks consumed. Just some weak reminder and reasons, and the limit is thus indeed 48.